src/vs/editor/contrib/suggest/suggestController.ts

```diff
@@ -423,7 +423,7 @@ const SuggestCommand = EditorCommand.bindToContribution<SuggestController>(Sugge
 registerEditorCommand(new SuggestCommand({
 	id: 'acceptSelectedSuggestion',
 	precondition: SuggestContext.Visible,
-	handler: x => x.acceptSelectedSuggestion(true),
+	handler: async (x) => { await x.acceptSelectedSuggestion(true); },
```

Comment:
Functions like this don't need to use async/await, the fat arrow function is already returning the promise

Reply:
But it's also fine to write it this way

Reply:
@roblourens Yea you're right. Originally, I was planning to make them all with async/await for clarity reasons, but I'm not sure I was consistent in that approach.